InstallModuleFromGitHub.psm1

```diff
@@ -24,7 +24,7 @@ function Install-ModuleFromGitHub {
         if($GitHubRepo) {
                 Write-Verbose ("[$(Get-Date)] Retrieving {0} {1}" -f $GitHubRepo, $Branch)
 
-                $url = "https://api.github.com/repos/{0}/zipball/{1}" -f $GitHubRepo, $Branch
+                $url = "https://api.github.com/repos{0}/zipball/{1}" -f $GitHubRepo, $Branch
```

Comment:
I think it'd be clearer to keep this line as-is and apply the fix in line 17 instead